core/frontend/src/extension/ExtensionAdmin.ts

```diff
@@ -0,0 +1,115 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Extensions
+ */
+
+import { IModelApp } from "../IModelApp";
+import { ActivationEvent, ExtensionManifest, LocalExtensionProps, ResolveFunc } from "./Extension";
+import { ExtensionLoader } from "./ExtensionLoader";
+
+/** The Extensions loading system has the following goals:
```

Comment:
Where if anywhere does this block of documentation show up in the published doc site?

Reply:
It does not today but it would be what we want to add as part of the learning documentation. It should be moved over there but those docs have yet to be written.